stl/inc/execution

```diff
@@ -26,9 +26,6 @@ _STL_DISABLE_CLANG_WARNINGS
 #pragma push_macro("new")
 #undef new
 
-// TRANSITION, Several names herein are declared as _Xxx_xxx2; in all such cases the corresponding

```

Comment:
For folks reading, these comments were transformed into tests that aren't yet present here in GitHub (I'm working on it right now, sorry!).

Reply:
For the folks wondering why it matters: ABI.

When we change the relationship between an internal function `_Fn` and public function `fn`, we have to change the name of `_Fn` to avoid "old" `fn` in already-compiled code "in the wild" linking to freshly-compiled code and consequently calling the "new" `_Fn` with catastrophic results. Ditto "new" `fn` must not call "old" `_Fn`. When the change touches the function parameters, generally the mangled name of `_Fn` is different. If the change doesn't alter the function parameters, we'll change the name of `_Fn` directly, say by adding a `2`. 

When the name is changed directly the old name `_Fn` becomes available. We could then inadvertently reuse it for something else, with potentially the same catastrophic results. Rather than simply keeping a list of names to avoid - which people would have to check when writing new code - we have a test that crashes if such a name is reused.